site/reference/cli/fn/eval/README.md

```diff
@@ -145,6 +146,12 @@ $ kpt fn eval DIR -i gcr.io/example.com/my-fn:v1.0.0 -- foo=bar
 $ kpt fn eval DIR --exec ./my-fn
 ```
 
+```shell
+# execute executable my-fn with arguments on the resources in DIR directory and
+# write output back to DIR
+$ kpt fn eval DIR --exec "./my-fn arg1 arg2"
```

Comment:
Could you update the book:

https://kpt.dev/book/05-developing-functions/03-developing-in-Typescript
https://kpt.dev/sdk/ts-guide

Reply:
done.